server/src/controllers/cards.controller.ts

```
@@ -37,4 +41,113 @@ const performanceStatus = asyncHandler(async (req: Request, res: Response) => {
+      .json(
+        new ApiResponse(
+          200,
+          { stats: result },
+          "Review Card Details fetch successfully.",
+        ),
+      );
+  },
+);
+
+const goalCardStats = asyncHandler(async (req: Request, res: Response) => {
+  const userId = new Types.ObjectId(req.user?.id!);
```

Biome (2.3.13)

[error] Forbidden non-null assertion after optional chaining.

Optional chaining already handles nullish values. Using non-null assertion defeats its purpose and may cause runtime errors.
Consider using the nullish coalescing operator `??` or optional chaining throughout the chain instead.

(lint/suspicious/noNonNullAssertedOptionalChain)